Homepwner/Base.lproj/Main.storyboard

```diff
@@ -165,13 +157,13 @@
                                         </subviews>
                                     </stackView>
                                     <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="249" text="Date Created" textAlignment="center" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="8mH-m2-jVY">
-                                        <rect key="frame" x="0.0" y="78" width="355" height="20.5"/>
+                                        <rect key="frame" x="0.0" y="114" width="375" height="20.5"/>
                                         <fontDescription key="fontDescription" type="system" pointSize="17"/>
                                         <nil key="textColor"/>
                                         <nil key="highlightedColor"/>
                                     </label>
-                                    <imageView userInteractionEnabled="NO" contentMode="scaleAspectFit" horizontalHuggingPriority="251" verticalHuggingPriority="248" verticalCompressionResistancePriority="751" translatesAutoresizingMaskIntoConstraints="NO" id="rVV-gR-rWF">
-                                        <rect key="frame" x="0.0" y="98.5" width="355" height="444.5"/>
+                                    <imageView userInteractionEnabled="NO" contentMode="scaleAspectFit" horizontalHuggingPriority="251" verticalHuggingPriority="248" verticalCompressionResistancePriority="749" translatesAutoresizingMaskIntoConstraints="NO" id="rVV-gR-rWF">
```

Comment:
This was the magic line that fixed the image. 

Still not sure how to get a minimum of 8 points between the stack views and date label. Will speak with you tomorrow about it. Thanks again for checking this project out!